File: 백준/Bronze/5063. TGN/TGN.cc
#include <iostream>
using namespace std;

int main() {
	cin.tie(0)->sync_with_stdio(0);

	int n;
	cin >> n;
	while (n--) {
		int r, e, c;
		cin >> r >> e >> c;
		if (r < e - c) cout << "advertise" << '\n';
		else if (r == e - c) cout << "does not matter" << '\n';
		else cout << "do not advertise" << '\n';
	}
}

File: 백준/Bronze/5063. TGN/TGN.py
import sys


def input():
    return sys.stdin.readline()


n = int(input())
for _ in range(n):
    r, e, c = map(int, input().split())
    if r < e - c:
        print('advertise')
    elif r == e - c:
        print('does not matter')
    else:
        print('do not advertise')
